packages/reporter/src/commands/command.tsx

```diff
@@ -100,11 +117,38 @@ const Aliases = observer(({ model, aliasesWithDuplicates }: AliasesProps) => {
 
 const Message = observer(({ model }) => (
   <span>
-    <i className={`fas fa-circle ${model.renderProps.indicator}`}></i>
-    <span
-      className='command-message-text'
-      dangerouslySetInnerHTML={{ __html: formattedMessage(model.displayMessage) }}
-    />
+    <span className='command-message-text-wrap'>
+      <i className={`fa fa-circle ${model.renderProps.indicator}`}></i>
+      <span
+        className='command-message-text'
+        dangerouslySetInnerHTML={{ __html: formattedMessage(model.displayMessage) }}
+      />
+    </span>
+    { model.options && Object.keys(model.options).length > 0
+      ?
+      <span className="command-message-options" onClick={(e) => e.stopPropagation()}>
+        <ObjectInspector
```

Comment:
We'll this automatically expanded when the tests are running outside of interactive mode. If a user is watching the video or looking at a screenshot of the test run, the options being collapsed will not be helpful.

Reply:
It's currently a bit hard because `reporter` doesn't have things like `Cypress.config` now. We need to find way to implement it.